parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -239,18 +239,14 @@ configureCNIIPTables() {
 disable1804SystemdResolved() {
     ls -ltr /etc/resolv.conf
     cat /etc/resolv.conf
-    {{- if Disable1804SystemdResolved}}
     UBUNTU_RELEASE=$(lsb_release -r -s)
-    if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
+    if [[ ${UBUNTU_RELEASE} == "18.04" || ${UBUNTU_RELEASE} == "20.04" ]]; then
```

Comment:
!= 16.04 would probably be better, as we support nothin < 16

Reply:
right, however, I had a problem with Mariner for != 16.04 when I was updating chrony, I will merge this now(to start the weekly release) and revisit later in case this grows. hope that is ok.